inc/TRestRawVetoAnalysisProcess.h

```diff
@@ -74,8 +74,8 @@ class TRestRawVetoAnalysisProcess : public TRestEventProcess {
 
    protected:
    public:
-    any GetInputEvent() const override { return fSignalEvent; }
-    any GetOutputEvent() const override { return fSignalEvent; }
+    RESTValue GetInputEvent() const override { return fSignalEvent; }
```

Comment:
I don't fully understand these (nor the original or the changes). Wouldn't they cause conflicts? Why not just use `TRestEvent*` here?

Reply:
I don't understand very well what is going on here but the base class `TRestEventProcess` is using `RESTValue` instead of `any`, which seems the origin of the issue. `RESTValue` is defined on `TRestReflector`:
https://github.com/rest-for-physics/framework/blob/cad41cda2051ee5c62f35a1e856518d567324bdd/source/framework/tools/inc/TRestReflector.h#L422

Perhaps the issue is this line:
https://github.com/rest-for-physics/framework/blob/cad41cda2051ee5c62f35a1e856518d567324bdd/source/framework/tools/inc/TRestReflector.h#L421
Since the use of a variable called `any` can be interpreted as `std::any` by the root interpreter.

I think we should move to `RESTValue` or `TRestEvent`, perhaps the second has issues with the inheritance because later on you want to use the inherited class rather than the base class `TRestEvent`. Anyway, we should get rid of `typedef REST_Reflection::TRestReflector any;` I think this is a different issue since the error is fixed in this PR. Perhaps you can create an issue about this since I don't want to keep adding things to this PR.